server.js

```diff
@@ -26,22 +26,23 @@ function Forecast(day) {
 app.get('/weather', async (request, response) => {
 
   
-  const city_name = request.query.city_name;
   const lat = request.query.lat;
   const lon = request.query.lon;
 
   const weatherAPI = `https://api.weatherbit.io/v2.0/forecast/daily?key=${process.env.REACT_APP_WEATHER_API_KEY}&land=en&lat=${lat}&lon=${lon}&days=5`
 
+  
 
+const firstClimate = await axios.get(weatherAPI);
 
+  
 
 
 
-
-  if (firstClimate) {
-    const weatherArr = currentForecast.data.map(item => new Forecast(item));
+  try {
+    const weatherArr = firstClimate.data.data.map(day => new Forecast(day));
     response.send(weatherArr);
-  } else {
+  } catch (error) {
     response.status(555).send('submit a valid city name')
   }
```

Comment:
Hi there!

Feedback here: I see that the map is rendering so nice work! For this section, consider creating a function `getWeatherData`. Inside this function is where the try catch block will live. Inside the function:

- move in your lat, long and axios call
- If there is a firstClimate (I would consider renaming the variable to responseData, or weatherData), then use line 43 where you define the constant `weatherArr`.
- Then, send ` response.status(200).send(responseData);
- Consider changing 555 error to 500.
- Grab a TA to get you going on weather and movie data 👍